docs/CLI/manifest_config.mdx

```diff
@@ -111,7 +111,7 @@ Additionally, the latest version of `dockerd` recognizes OCI manifests and does
 ## Push / Pull config
 
 As mentioned in the beginning of this doc, the manifest config is not used by `oras` and is not worth pulling in most scenarios.
```

Comment:
I'm a little confused because this link seems to be working for me https://oras.land/docs/cli/manifest_config/

Reply:
Not for me https://oras.land/docs/cli/manifest_config/manifest_annotations

Reply:
Talking about this one 
![Screenshot_20230624-214948](https://github.com/oras-project/oras-www/assets/318490/13a8ae0e-924e-439e-b1b0-36db2e7c3e1e)

Reply:
Ah, works for https://oras.land/docs/CLI/manifest_config/ but not for https://oras.land/docs/CLI/manifest_config

Reply:
Doesn't work for me in either case 😕

Reply:
there is some odd url rewrite going on with https://oras.land/docs/CLI/manifest_config with a lower case "cli"

Reply:
Yeah, but both the links for me. I have updated the changes in PR #194. Thanks for noticing this @pdecat :D